src/modules/Modals.js

```diff
@@ -0,0 +1,33 @@
+export class Comment {
+  constructor(username, creation_date, comment) {
+    this.username = username;
+    this.creation_date = creation_date;
+    this.comment = comment;
+  }
+}
+
+export class Reserve {
+  constructor(username, date_start, date_end) {
+    this.username = username;
+    this.date_start = date_start;
+    this.date_end = date_end;
+  }
+}
+
+export class Launch {
+  likes;
+
+   reserves = [];
+
+  comments = [];
```

Comment:
- [OPTIONAL] I think it's better to keep on code just the necessary variables. You can keep these variables if your code needs them.